src/app/core/entity-components/entity-subrecord/entity-subrecord/entity-subrecord.component.spec.ts

```diff
@@ -133,10 +131,10 @@ describe("EntitySubrecordComponent", () => {
 
   it("should sort non-standard objects", () => {
```

Comment:
I think this should rather be a new test: sort for normal objects **and** sort with `_ordinal` value present.